bl.py

```diff
@@ -0,0 +1,171 @@
+    os.kill(os.getppid(), signal.SIGKILL)
+
+
+def create(name, templates=()):
+    """creates new project"""
+    for template in templates:
+        if not check_if_project_exist(template):
+            raise Exception("template: %s not found" % template)
+    if not os.path.isfile("./shell_profiles"):
+        Popen("./create_shell.sh")
+    if not os.path.exists("./projects/%s" % name):
+        os.makedirs("./projects/%s" % name)
+    if not os.path.exists("./projects/%s/tasks" % name):
+        os.makedirs("./projects/%s/tasks" % name)
+
+    with open("./projects/%s/%s.py" % (name, name), mode='w') as project_file:
+        project_file.write("""import click
+import bl
+
+
+@click.group(chain=True, invoke_without_command=True)
+@click.pass_context
+def cli(ctx):
+    if ctx.invoked_subcommand is None:
+        bl.start('%s')
+""" % name)
+
+    with open("./projects/%s/project_start" % name, mode='w') as start_file:
+        if templates:
+            start_file.write("# TEMPLATES\n")
+            for template in templates:
+                start_file.write("# from template: %s\n" % template)
+                template_start_file = open("./projects/%s/project_start" % template)
+                start_file.write(template_start_file.read())
+                start_file.write("\n")
+            start_file.write("# check if correctly imported templates\n")
+        else:
+            start_file.write('# add here shell code to be executed while entering project\n')
+    with open("./projects/%s/project_stop" % name, mode='w') as stop_file:
+        if templates:
+            stop_file.write("# TEMPLATES\n")
+            for template in templates:
+                stop_file.write("# from template: %s\n" % template)
+                template_stop_file = open("./projects/%s/project_stop" % template)
+                stop_file.write(template_stop_file.read())
+                stop_file.write("\n")
+            stop_file.write("# check if correctly imported templates\n")
+        else:
+            stop_file.write('# add here shell code to be executed while exiting project\n')
+    edit_file("./projects/%s/project_start" % name)
+    edit_file("./projects/%s/project_stop" % name)
+
+
+def create_task(project, name, description):
+    if check_if_project_exist(project):
+        if not check_if_task_exist(project, name):
+            Popen(["/bin/sh", "-c", "echo '#!/bin/sh' > ./projects/%s/tasks/%s.sh" % (project, name)]).communicate(input)
+            edit_file("./projects/%s/tasks/%s.sh" % (project, name))
+            os.chmod("./projects/%s/tasks/%s.sh" % (project, name), 0o755)
+            with open("./projects/%s/%s.py" % (project, project), mode='a') as project_file:
+                project_file.write("""
+
+@cli.command()
+@click.argument('args', nargs=-1)
+def %s(args=()):
+    \"""%s""\"
+    bl.run_task("%s", "%s", args)
+""" % (name, description, project, name))
+        else:
+            raise Exception("task already exists")
```

Comment:
maybe instead of having general exception.

Create `exceptions.py` with custom exceptions related to pet project. 
Reference - https://github.com/PythonicNinja/pydrill/blob/master/pydrill/exceptions.py